core/verify_modes.go

```diff
@@ -0,0 +1,69 @@
+package core
+
+import "fmt"
+
+type VerifyMode uint32
+
+const (
+	LocalVerify VerifyMode = iota //
+	FullVerify
+	LightVerify
+	InsecureVerify
```

Comment:
`InsecureVerify` here means no verification at all, I think it is better name as `SkipVerify`.

And for `LightVerify` which is actually not secure, it is better name as `InsecureVerify`